images.CI/macos/move-vm.ps1

```diff
@@ -48,12 +48,23 @@ Import-Module $PSScriptRoot\helpers.psm1 -DisableNameChecking
 # Connection to a vCenter Server system
 Connect-VCServer
 
-try
-{
-    Get-VM $VMName | Move-VM -Datastore $TargetDataStore -ErrorAction Stop
-    Write-Host "VM has been moved successfully to target datastore '$TargetDataStore'"
+$Vm = Get-VM $VMName
```

Comment:
$Vm -> $vm to consistent with other scripts

Reply:
changed